From 66d1d752976f4b45a5faff2bc68fc2fa9d435c71 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sun, 14 May 2017 22:04:55 +0200
Subject: [PATCH] utils/qv4l2/Makefile.am: link with librt

qv4l2 uses clock_gettime(), so it should link with librt to be
compatible with old versions of glibc.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 utils/qv4l2/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/qv4l2/Makefile.am b/utils/qv4l2/Makefile.am
index fd58486..a991d66 100644
--- a/utils/qv4l2/Makefile.am
+++ b/utils/qv4l2/Makefile.am
@@ -7,7 +7,7 @@ qv4l2_SOURCES = qv4l2.cpp general-tab.cpp ctrl-tab.cpp vbi-tab.cpp capture-win.c
   v4l2-tpg-core.c v4l2-tpg-colors.c
 nodist_qv4l2_SOURCES = moc_qv4l2.cpp moc_general-tab.cpp moc_capture-win.cpp moc_vbi-tab.cpp qrc_qv4l2.cpp
 qv4l2_LDADD = ../../lib/libv4l2/libv4l2.la ../../lib/libv4lconvert/libv4lconvert.la \
-  ../libv4l2util/libv4l2util.la ../libmedia_dev/libmedia_dev.la
+  ../libv4l2util/libv4l2util.la ../libmedia_dev/libmedia_dev.la -lrt
 qv4l2_CPPFLAGS = -I../common
 
 if WITH_QTGL
-- 
2.7.4

